CONTRIBUTING.rst

```diff
@@ -47,6 +47,7 @@ Tests are in ``rst2pdf/tests`` directory. Within this folder:
   - ``test_[name].txt`` holds the rst source for the test (yes, really)
   - ``test_[name].style`` has any stylesheet that should be applied for this test
   - ``test_[name].cli`` allows adding extra commands to the test (look for examples)
+  - ``test_[name].exitcode`` allows for specifying an expected non-zero exit status code for expected error cases
```

Comment:
It's a nit, but can we call this retcode? That seems to be more common

Reply:
Done.